frontend/packages/core/src/Input/select.tsx

```diff
@@ -178,6 +179,7 @@ const StyledSelect = styled(BaseSelect)({
 interface SelectOption {
   label: string;
   value?: string;
+  startAdornment?: React.ReactElement;
```

Comment:
would be nice to have this example on storybook but doesn't have to be done in this PR